src/clusterfuzz/_internal/bot/tasks/trials.py

```diff
@@ -43,9 +43,10 @@ def __init__(self):
     for extension in extensions_to_strip:
       app_name = utils.strip_from_right(app_name, extension)
 
-    self.trials = {}
     for trial in data_types.Trial.query(data_types.Trial.app_name == app_name):
-      self.trials[trial.app_args] = trial.probability
+      self.trials[trial.app_args] = {'probability': trial.probability}
+      if hasattr(trial, 'contradicts'):
```

Comment:
Rather than do this, can you modify https://github.com/google/clusterfuzz/blob/770bee181da6f309c05b424053e1b7bb4b4b340b/src/clusterfuzz/_internal/datastore/data_types.py#L1382 to add this property to the Model?

Reply:
We can do this, but note that V8 isn't going to use the in-DB trials anymore. They are legacy from our viewpoint and only have few chrome entries. Not sure who maintains them. From our viewpoint we could keep any new features out of the model, but the code might indeed be cleaner by supporting it without much cost...

Reply:
Note also, if we don't add it to the model, these two lines are dead code.

Reply:
Re on simple: I wouldn't want to add another table to get a list of flags. It'd do to create a string property and just space separate flags. We sort-of have the convention anyways to not have spaces within one flag.

Reply:
done